panoptes_aggregation/batch_aggregation.py

```diff
@@ -0,0 +1,156 @@
+from celery import Celery
+import json
+import pandas as pd
+import os
+import urllib3
+from shutil import make_archive
+import uuid
+
+from azure.storage.blob import BlobServiceClient
+
+from panoptes_client import Panoptes, Project, Workflow
+from panoptes_aggregation.workflow_config import workflow_extractor_config
+from panoptes_aggregation.scripts import batch_utils
+
+import logging
+panoptes_client_logger = logging.getLogger('panoptes_client').setLevel(logging.ERROR)
+
+celery = Celery(__name__)
+celery.conf.broker_url = os.environ.get("CELERY_BROKER_URL", "redis://localhost:6379")
+celery.conf.result_backend = os.environ.get("CELERY_RESULT_BACKEND", "redis://localhost:6379")
+
+
+@celery.task(name="run_aggregation")
+def run_aggregation(project_id, workflow_id, user_id):
+    ba = BatchAggregator(project_id, workflow_id, user_id)
+    ba.save_exports()
+
+    ba.process_wf_export(ba.wf_csv)
+    cls_df = ba.process_cls_export(ba.cls_csv)
+
+    extractor_config = workflow_extractor_config(ba.tasks)
+    extracted_data = batch_utils.batch_extract(cls_df, extractor_config)
+
+    batch_standard_reducers = {
+        'question_extractor': ['question_reducer', 'question_consensus_reducer'],
+        'survey_extractor': ['survey_reducer']
+    }
+
+    for task_type, extract_df in extracted_data.items():
+        extract_df.to_csv(f'{ba.output_path}/{ba.workflow_id}_{task_type}.csv')
+        reducer_list = batch_standard_reducers[task_type]
+        reduced_data = {}
+
+        for reducer in reducer_list:
+            # This is an override. The workflow_reducer_config method returns a config object
+            # that is incompatible with the batch_utils batch_reduce method
```

Comment:
I used those as a starting point. The goal was to have multiple reducers run in certain cases (like question & question consensus). The `batch_standard_reducers` dict can be moved wherever makes sense later. 

This override is because the output of `workflow_config.workflow_reducer_config` currently causes an error when passed to `batch_utils.parse_reducer_config`. Possibly a bug, or just a discrepancy between the function and yaml outputs. Since batch_utils is now used to process csvs, making changes to batch_utils felt out of scope for this PR.